src/components/Exchange/NetValueTooltip.tsx

```diff
@@ -0,0 +1,64 @@
+import { Trans, t } from "@lingui/macro";
+import StatsTooltipRow from "components/StatsTooltip/StatsTooltipRow";
+import Tooltip from "components/Tooltip/Tooltip";
+import { BigNumberish } from "ethers";
+import { USD_DECIMALS } from "lib/legacy";
+import { formatAmount } from "lib/numbers";
+
+type Props = {
+  position: {
```

Comment:
might be better to define Position type somewhere and reuse it to avoid code duplication, do you think we can do it in this PR?

Reply:
not critical though

Reply:
I have added the type. Please have a look.